zrml/swaps/src/tests.rs

```diff
@@ -3080,8 +3080,6 @@ fn assert_all_parameters(
     assert_eq!(Currencies::free_balance(ASSET_B, &pai), pool_assets[1]);
     assert_eq!(Currencies::free_balance(ASSET_C, &pai), pool_assets[2]);
     assert_eq!(Currencies::free_balance(ASSET_D, &pai), pool_assets[3]);
-
-    assert_eq!(Currencies::total_issuance(psi), total_issuance);
```

Comment:
We mustn't remove that check, many tests rely on that. To fix the issue we have to fix the corresponding test (line 141+):
```rust
#[test]
fn destroy_pool_correctly_cleans_up_pool() {
    ExtBuilder::default().build().execute_with(|| {
        create_initial_pool_with_funds_for_alice(ScoringRule::CPMM, Some(0), true);
        let pool_id = 0;
        let alice_balance_before = [
            Currencies::free_balance(ASSET_A, &ALICE),
            Currencies::free_balance(ASSET_B, &ALICE),
            Currencies::free_balance(ASSET_C, &ALICE),
            Currencies::free_balance(ASSET_D, &ALICE),
        ];
        assert_ok!(Swaps::destroy_pool(pool_id));
        assert_err!(Swaps::pool(pool_id), crate::Error::<Runtime>::PoolDoesNotExist);
        // Ensure that funds _outside_ of the pool are not impacted!
        // TODO(#792): Remove pool shares.
        let total_pool_shares = Currencies::total_issuance(Swaps::pool_shares_id(0));
        assert_all_parameters(alice_balance_before, 0, [0, 0, 0, 0], total_pool_shares);
    });
}
```
I just changed the last three lines of that function